dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java

```diff
@@ -170,6 +171,6 @@ public void configClientPipeline(URL url, ChannelPipeline pipeline, SslContext s
             .build();
         final Http2MultiplexHandler handler = new Http2MultiplexHandler(
             new TripleClientHandler(frameworkModel));
-        pipeline.addLast(codec, handler, new TripleTailHandler());
+        pipeline.addLast(new TelnetHeaderConsumer(), codec, handler, new TripleTailHandler());
```

Comment:
why add it here？

Reply:
For a telnet connection,  pu server need to send prompt message to the client(pu server don't know client protocol type, may triple, may telnet). If client is a triple client, this decoder is responsible for consume prompt message which is useless for triple client(or will cause wrong tcp stream unpack).  

- I am investigating telnet protocol and telnet command so that client could send some message to server once connected.

Reply:
why `pu server need to send prompt message to the client`?